data/data/azure/dns/dns.tf

```diff
@@ -0,0 +1,28 @@
+resource "azurerm_dns_zone" "private" {
+  name                           = "${var.base_domain}"
+  resource_group_name            = "${var.resource_group_name}"
+  zone_type                      = "Private"
+  resolution_virtual_network_ids = ["${var.internal_dns_resolution_vnet_id}"]
+}
+
+resource "azurerm_dns_zone" "public" {
+  name                = "${var.cluster_domain}"
+  resource_group_name = "${var.resource_group_name}"
+  zone_type           = "Public"
+}
+
+resource "azurerm_dns_a_record" "api_internal" {
+  name                = "api.${var.cluster_name}"
+  zone_name           = "${azurerm_dns_zone.private.name}"
+  resource_group_name = "${var.resource_group_name}"
+  ttl                 = 300
+  records             = ["${var.internal_lb_ipaddress}"]
```

Comment:
does this mean our internal ip addresses for lb will never change?

Reply:
correct, the lb ip addressing is set as Static https://github.com/openshift/installer/blob/9a60194256a9469602406c3548fa6a24519fbb85/data/data/azure/vnet/internal-lb.tf#L10